System/Process/Internals.hs

```diff
@@ -200,22 +200,22 @@ runGenProcess_ fun c _ _ = createProcess_ fun c
 --
 -- @
 --     #if defined(__IO_MANAGER_WINIO__)
---     import GHC.IO.SubSystem ((<!>))
+--     import GHC.IO.SubSystem ((`<!>`))
 --     import GHC.IO.Handle.Windows (handleToHANDLE)
 --     import GHC.Event.Windows (associateHandle')
 --     #endif
 --
 --     ...
 --
---     #if defined (__IO_MANAGER_WINIO__)
---     return () <!> (do
---       associateHandle' =<< handleToHANDLE <handle>)
+--     #if defined(__IO_MANAGER_WINIO__)
+--     return () `<!>` do
+--       associateHandle' `=<<` handleToHANDLE readEnd
```

Comment:
Would you mind posting a before and after screenshot? I'm confused what the backticks are for (escaping the `<`?).

Reply:
> I'm confused what the backticks are for (escaping the `<`?).

They hyperlink the identifier.   The main reason I used them here is that otherwise `<` would need to be escaped as `\<`.   That they also hyperlink  is a nice by product.

> Would you mind posting a before and after screenshot?

Sorry, I'm away from my computer right now.

Please look at Hackage for how it is currently rendered.  You will immediately see that it is broken. This PR fixes this. You can verify this with `cabal haddock`.

Reply:
OK, sure. I don't think we need to hyperlink `=<<`, and the link didn't appear on `<!>` anyway, so I converted it to escaping.

### Before

![screenshot](https://github.com/user-attachments/assets/cf91a23a-276f-4409-a6d1-3a3e1090547a)

### After

![screenshot2](https://github.com/user-attachments/assets/359838b6-e298-4aa0-a58a-89f50ea64ec7)